pontoon/base/templates/django/base.html

```diff
@@ -43,7 +44,12 @@
         <li><a href="https://developer.mozilla.org/en-US/docs/Localizing_with_Pontoon" target="_blank">Help</a></li>
         <li id="admin"{% if not perms.base.can_manage %} class="hidden"{% endif %}><a href="{% url "pontoon.admin" %}">Admin</a></li>
         {% if not user.is_authenticated %}
-        <li id="sign-in"><a href="{% url 'pontoon.persona-migration' %}">Sign in</a></li>
+        <li id="sign-in">
+            Sign in via:
+            <ul>
+                <li><a href="{% provider_login_url 'fxa' scope='profile:uid profile:email profile:display_name' %}">Firefox Accounts</a></li>
```

Comment:
Is thre a reason pass scope explicitly? django-allauth should load the default scope from settings.